agent/requirements.txt

```diff
@@ -7,6 +7,6 @@ jinja2
 python-daemon
 python-pidfile
 redis
-requests>=2.31.0  # CVE-2023-32681
+requests>=2.27.0  # JSONDecodeError, TODO CVE-2023-32681 (2.31)
```

Comment:
Would it resolve the situation if we omitted the minimum requirement from here and put a cap in [`agent/test-constraints-3.6.txt`](https://github.com/distributed-system-analysis/pbench/blob/d302664f0861d5865b9cc4471db0a0ebd2c7fec7/agent/test-constraints-3.6.txt#L1) instead?

Reply:
Not quite; the reason I touched `requirements.txt` in the first place was that my RHEL 8.7 system met the unqualified requirement with requests 2.25; and the agent has a (previously unexpressed) dependency on requests 2.27, which introduced the `requests.exceptions.JSONDecodeError` exception. And `test-constraints.txt` is only for the tests, not for the product build, anyway.